### PR TITLE
New types for rpc calls

### LINKED ISSUE #313: All the data from the server need Types
The App types for all the Info from the server...

### PR DESCRIPTION
**Pull request by juanky201271**
I created a few new types for all the RPC calls.

### CODE REVIEW

File: app/uris/parseZcashURI.ts [hunk 1]
```diff
@@ -19,9 +20,16 @@ const parseZcashURI = async (uri: string): Promise<string | ZcashURITarget> => {
   // The first address is special, it can be the "host" part of the URI
   const address = parsedUri.pathname;
 
-  const resultParse = await RPCModule.execute('parse', address);
-  //console.log('parse', resultParse);
-  const resultParseJSON = await JSON.parse(resultParse);
+  const resultParse: string = await RPCModule.execute('parse', address);
+  if (resultParse) {
+    if (resultParse.toLowerCase().startsWith('error')) {
+      return 'Right now it is not possible to verify the address with the server';
```
Comment (fluidvanadium) [inline app/uris/parseZcashURI.ts:30]: this error message is unclear

**Review by fluidvanadium — CHANGES_REQUESTED** [review]
i like the way that these results are parsed more carefully. im not sure about the error message

File: app/uris/parseZcashURI.ts (hunk 1 above)
Comment (juanky201271) [reply app/uris/parseZcashURI.ts:30]: I'm all ears, about some advice of the native English speakers... @zancas @AloeareV

Comment (juanky201271) [reply app/uris/parseZcashURI.ts:30]: two days later, some idea about the perfect message here?

Comment (juanky201271) [reply app/uris/parseZcashURI.ts:30]: @Oscar-Pepper can you help us with a more clear message?

**Review by fluidvanadium — APPROVED** [review]

**Pull request merged**